coalib/settings/Setting.py

```diff
@@ -1,4 +1,8 @@
+#  Copyright (c) Django Software Foundation and individual contributors.
+#  License: https://raw.githubusercontent.com/django/django/master/LICENSE
```

Comment:
@Makman2 How to comply to the license?
https://github.com/coala-analyzer/coala/pull/1891/files#r56218492
I thought that to put the copyright and license file link would be compyling to the license.
No idea how licensing works